src/cap32.cpp

```diff
@@ -3347,7 +3348,7 @@ int cap32_main (int argc, char **argv)
       exit(-1);
    }
 
-   loadConfiguration(CPC, getConfigurationFilename()); // retrieve the emulator configuration
+   loadConfiguration(CPC, args.cfgFilePath != "" ? args.cfgFilePath : getConfigurationFilename()); // retrieve the emulator configuration
```

Comment:
Fixed. And yes it does work well if the file does not exist: no error reported, and you get a default configuration.